kythe/docs/schema/schema.txt

```diff
@@ -610,7 +610,8 @@ and target languages do not share Kythe indexers, it is still semantically
 useful to connect the nodes they emit. For example, one might want to list
 all the $$C++$$ and Java uses of a particular service call, starting at the
 specification of that service. The specification and its generated artifacts
-may be joined by the *generates* edge.
+may be joined by the *generates* edge.  Both semantic nodes and their enclosing
```

Comment:
I think that files can generate files and semantic nodes can generate semantic nodes, but I'm not sure if we should relate files with semantic nodes in this way.